src/test/java/net/kyori/adventure/text/minimessage/MiniMessageParserTest.java

```diff
@@ -116,6 +117,24 @@ public void testEscapeInner() {
     assertEquals(expected, MiniMessageParser.escapeTokens(input));
   }
 
+  @Test
+  public void testUnescape() {
+    final String input ="<yellow>TEST\\<green\\> nested\\</green\\>Test";
+    final String expected = "TEST<green> nested</green>Test";
+    TextComponent comp = (TextComponent) MiniMessageParser.parseFormat(input);
+
+    assertEquals(expected, comp.content());
+  }
+
+  @Test
+  public void testNoUnescape() {
+    final String input ="<yellow>TEST\\<green\\>\\> \\< nested\\</green\\>Test";
+    final String expected = "TEST<green>\\> \\< nested</green>Test";
+    TextComponent comp = (TextComponent) MiniMessageParser.parseFormat(input);
+
+    assertEquals(expected, comp.content());
+  }
+
 
```

Comment:
should just be a single empty line here

Reply:
I may have misinterpreted some other double-line-breaks in the files as having meaning :D

Reply:
ah yeah -- no minimessage just hasn't been properly checkstyled yet, so the formatting is still kinda inconsistent.

Reply:
Fixed! :)